src/main/java/frc/robot/hardware/digitalinput/DigitalInputInputs.java

```diff
@@ -0,0 +1,11 @@
+package frc.robot.hardware.digitalinput;
+
+import org.littletonrobotics.junction.AutoLog;
+
+@AutoLog
+public class DigitalInputInputs {
+
+	public boolean debouncedValue = false;
+	public boolean nonDebouncedValue = false;
```

Comment:
If `nonDebouncedValue` is the negation of `debouncedValue`, why is it a public variable? Shouldn't it be handled privately based on negating the debounced value? If this is not the case, please give this variable a more descriptive name.